migrations/20180725202700-populate-collective-with-IRS-Bot.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const insert = (sequelize, table, entry) => {
+  return sequelize.query(`
+    INSERT INTO "${table}" ("${Object.keys(entry).join('","')}") VALUES (:${Object.keys(entry).join(",:")})
+  `, { replacements: entry });
+};
+
+module.exports = {
+  up: (queryInterface, sequelize) => {
+    
+    // Inserting OpenCollective Bot Collective
+    const botData = {
+      name: 'IRS Bot',
+      mission: 'Support users through the platform',
+      description: 'IRS bot that support users regarding IRS issues through the platform',
+      longDescription: 'IRS bot that support users regarding IRS issues  through the platform',
+      currency: 'USD',
+      image: 'https://cldup.com/rdmBCmH20l.png',
+      isActive: true,
+      slug: 'irs-bot',
+      website: 'https://opencollective.com',
+      type: 'BOT',
+      settings: JSON.stringify({
+        thresholdW9: 60000,
+        thresholdW9HtmlComment: '<p>You have now been paid $600 or more through Open Collective, which' +
```

Comment:
They have not been paid $600 or more yet.
You can say instead: 
> The total amount of the expenses that you have submitted this year to this host exceeds $600. To comply with the IRS, we need you to send us a <a href="https://www.irs.gov/pub/irs-pdf/fw9.pdf">W9 form</a> if you are a US resident (or a <a href="https://www.irs.gov/pub/irs-pdf/fw8ben.pdf">W9-BEN form</a> if you are not a US resident) before we can proceed with this payment. <a href="https://github.com/opencollective/opencollective/wiki/Submitting-Expenses#taxes">More info on our wiki</a>.

Reply:
There is also the [Form W-8BEN-E](https://www.irs.gov/pub/irs-pdf/fw8bene.pdf) for Non-US companies, right? so I believe we should do something like:

> The total amount of the expenses that you have submitted this year to this host exceeds $600. To comply with the IRS, we need you to send us a <a href="https://www.irs.gov/pub/irs-pdf/fw9.pdf">W9 form</a> if you are a US resident (if you are not a US resident send <a href="https://www.irs.gov/pub/irs-pdf/fw8ben.pdf">W-8BEN form</a> for individuals or <a href="https://www.irs.gov/pub/irs-pdf/fw8bene.pdf">W-8BEN-E form</a> for companies ) before we can proceed with this payment. <a href="https://github.com/opencollective/opencollective/wiki/Submitting-Expenses#taxes">More info on our wiki</a>.